micrometer/src/test/java/feign/micrometer/MicrometerCapabilityTest.java

```diff
@@ -18,19 +18,96 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Function;
 import java.util.stream.Collectors;
-import feign.Capability;
-import feign.Util;
+import feign.*;
+import feign.mock.HttpMethod;
+import feign.mock.MockClient;
+import feign.mock.MockTarget;
 import io.micrometer.core.instrument.Meter;
 import io.micrometer.core.instrument.Meter.Id;
 import io.micrometer.core.instrument.MockClock;
 import io.micrometer.core.instrument.simple.SimpleConfig;
 import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.junit.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertThrows;
 
 public class MicrometerCapabilityTest
     extends AbstractMetricsTestBase<SimpleMeterRegistry, Id, Meter> {
 
+
+  public interface SourceWithPathExpressions {
+
+    @RequestLine("GET /get/{id}")
+    String get(@Param("id") String id, String body);
+
+  }
+
+  @Test
+  public void clientMetricsHaveUriLabel() {
```

Comment:
Can we move this tests to the `AbstractMetricsTestBase`, likely this will fail for `dropwizard-metrics` which would need to be made consistent.

Reply:
I see - I originally placed it in `AbstractMetricsTestBase`, however I realised that `dropwizard-metrics` don't attach `uri` and thought that was a design decision. If necessary I can add `uri` on dropwizard 4 and 5 - is that what we would like? Thanks!

Reply:
I have moved the tests in `AbstractMetricsTestBase` and made `dropwizard-metrics` 4 & 5 consistent :)